data_structures/src/transaction.rs

```diff
@@ -259,6 +279,29 @@ impl DRTransaction {
             .map(|tx_idx| TxInclusionProof::new(tx_idx, txs))
     }
 
+    /// Returns the weight that a data request transaction.
+    /// This is the weight that will be used to calculate
+    /// how many transactions can fit inside one block
+    pub fn weight(&self) -> u32 {
+        // DR_weight = DR_size*alpha + W*COMMIT + W*REVEAL*beta + TALLY*beta + W*OUTPUT_SIZE
+
+        let dr_size = u32::try_from(self.to_pb().write_to_bytes().unwrap().len()).unwrap();
```

Comment:
Shouldn't this be the size of the data request only, not the data request transaction?

```suggestion
        let dr_size = u32::try_from(self.dr_output.to_pb().write_to_bytes().unwrap().len()).unwrap();
```